uber/templates/dept_checklist/hotel_requests.html

```diff
@@ -2,17 +2,6 @@
 {% block title %}Hotel Requests{% endblock %}}
```

Comment:
@kitsuta: I realize that this isn't directly related to this PR, but am I correct that this line has a typo?  It looks like there's an extra closing } character.

Reply:
Oh wow, it does!